api/v1/vector_store.proto

```diff
@@ -0,0 +1,223 @@
+syntax = "proto3";
+
+package llmoperator.vectore_store.v1;
+
+import "google/api/annotations.proto";
+
+option go_package = "github.com/llm-operator/file-manager/api/v1";
+
+// The API specification follows OpenAPI API specification (https://platform.openai.com/docs/api-reference/vector-stores).
+
+message ExpiresAfter {
+    // Anchor timestamp after which the expiration policy applies. Supported anchors: last_active_at.
+    string anchor = 1;
+    // The number of days after the anchor time that the vector store will expire.
+    int64 days = 2; 
+}
+
+message VectorStore{
+    string id = 1;
+    string object = 2;
+    // The Unix timestamp (in seconds) for when the vector store was created.
+    int64 created_at = 3;
+    string name = 4;
+    // The total number of bytes used by the files in the vector store.
+    int64 usage_bytes = 5;
+    int64 file_counts = 6;
+    // The status of the vector store, which can be either expired, in_progress, or completed. 
+    // A status of completed indicates that the vector store is ready for use.
+    string status = 7;
+    ExpiresAfter expires_after = 8;
+    // integer or null. The Unix timestamp (in seconds) for when the vector store will expire.
+    int64 expires_at = 9;
+    // integer or null. The Unix timestamp (in seconds) for when the vector store was last active.
+    int64 last_active_at = 10;
+    map<string, string> metadata = 11;
+}
+
+message AutoChunkingStrategy {
+    string type = 1;
+}
+
+message StaticChunkingStrategy {
+    string type = 1;
+    message Static {
+        int64 max_chunk_size_tokens = 1;
+        int64 chunk_overlap_tokens = 2;
+    }
+    Static statis = 2;
+}
+
+message OtherChunkingStrategy {
+    string type = 1;
+}
+
+message ChunkingStraty {
+    AutoChunkingStrategy auto_chunking_strategy = 1;
+    StaticChunkingStrategy static_chunking_strategy = 2;
+    OtherChunkingStrategy other_chunking_strategy = 3;
+}
+
+message CreateVectorStoreRequest {
+    repeated string file_ids = 1;
+    string name = 2;
+    ExpiresAfter expires_after = 3;
+    ChunkingStraty chunking_strategy = 4;
+    map<string, string> metadata = 5;
+}
+
+message ListVectorStoresRequest {
```

Comment:
FYI: When we implement a pagination, we can follow the code in job manager.

https://github.com/llm-operator/job-manager/blob/main/server/internal/server/jobs.go#L155